test/api/connectors/ConnectorSpec.scala

```diff
@@ -211,7 +210,7 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
     MockedAppConfig.api1661Environment returns "api1661-environment"
     MockedAppConfig.api1661EnvironmentHeaders returns Some(allowedIfsHeaders)
 
-    MockedAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> false)
+    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(true).anyNumberOfTimes()
```

Comment:
Was false before; should be false now:
```suggestion
    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(false).anyNumberOfTimes()
```